src/Components/Movies.js

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+
+const Movies = () => {
+  const [movies, setMovies] = useState([]);
+  const [title, setTitle] = useState("");
```

Comment:
You can just have one piece of state that holds the movie object and then you can access title, release date, and description from there